docs/template/public/less/_side-nav.less

```diff
@@ -42,10 +42,11 @@
        
         &:hover {
             color: @blue_gd;
+            background-color: transparent;
         }
     }
 
-    .kss-active a {
```

Comment:
@saadmhybris  is there a need to change it from css-active to active? This change is going to call for a full regression check of every page